include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -40,11 +40,14 @@ struct __subgroup_radix_sort
         using __block_size_t = ::std::integral_constant<::std::uint16_t, __block_size>;
         using __call_0_t = ::std::integral_constant<::std::uint16_t, 0>;
         using __call_1_t = ::std::integral_constant<::std::uint16_t, 1>;
+        using __call_2_t = ::std::integral_constant<::std::uint16_t, 2>;
 
         using _SortKernelLoc = oneapi::dpl::__par_backend_hetero::__internal::__kernel_name_provider<
             __radix_sort_one_wg_kernel<_KernelNameBase, __wg_size_t, __block_size_t, __call_0_t>>;
-        using _SortKernelGlob = oneapi::dpl::__par_backend_hetero::__internal::__kernel_name_provider<
+        using _SortKernelPartGlob = oneapi::dpl::__par_backend_hetero::__internal::__kernel_name_provider<
```

Comment:
What does `Part` mean here? is that partial? If so, do we have Partial Global?

Reply:
Here `Part` means that temporary storage within the algorithm is partially stored in global memory and partially in shared local memory. In particular, only the counters are stored in SLM here. The other two cases either place temporary storage entirely in global memory or entirely in SLM.

If there is a need to change this naming, I can open a second PR to rename this.